alerts/google-quotas/metadata.yaml

```diff
@@ -4,4 +4,23 @@ alert_policy_templates:
   display_name: Quota Exceeded Alert Policy
   description: "Alert policy for exceeding quota limit"
   version: 1
-
+-
+  id: single-allocation-quota
+  display_name: High quota utilization
+  description: "Monitoring quota utilization and will notify you if the utilization rises above alert trigger threshold."
```

Comment:
the description for the newly added templates are all the same currently. would it be easier to understand if more details are included so users can differentiate?

Reply:
Thanks.  Yes.  I've updated the description to match the templates they are describing.